Makefile

```diff
@@ -25,112 +26,144 @@ unittest: unittest-dev-241
 unittest-dev-222:
 	@echo "Running unittests"
 	@sudo rm -rf /home/ansys/.local/share/ansys_fluent_core/examples
-	@pip install -r requirements/requirements_tests.txt
+	@pip uninstall poetry -y
+	@pip install poetry>=1.8.2
```

Comment:
Is it better to remove this repetation with a requirements_build.txt?

Reply:
The main issue that even after installing poetry twice in `Install pyfluent` and `Install again after codegen` still we are getting following error in each unit test check.

![image](https://github.com/ansys/pyfluent/assets/106588300/9e0a7f25-08bd-49af-ac56-4caadae2168c)

To resolve above error we added above duplication otherwise we don't need those 2 lines each time.

Reply:
If we are going to retain requirements_*.txt then use of poetry will not serve it's purpose fully because I think it replaces setup.py, requirements_*.txt.

Reply:
Yes, poetry needs to be installed separately in the VM that is executing the unittests, because poetry is not a dependency of PyFluent. I meant to replace the above lines in all places with `pip install -r requirements/requirements_build.txt`. `requirements_build.txt` would contain `poetry>=1.8.2`.

Reply:
Done. Thank you.